ai_flow/api/ops.py

```diff
@@ -516,7 +516,8 @@ def action_on_model_version_event(job_name: Text,
                                   model_name: Text,
                                   model_version_event_type: Text,
                                   namespace: Text = DEFAULT_NAMESPACE,
-                                  action: JobAction = JobAction.RESTART):
+                                  action: JobAction = JobAction.RESTART,
+                                  sender: Text = ANY_CONDITION):
```

Comment:
IMO, the `sender` parameter is confusing for user. Because the user may not know that the model version event is sent by the Model Center service, which will cause that the user doesn't know how to set the value of the sender.